core/impl-base/src/main/java/org/jboss/arquillian/core/impl/ManagerImpl.java

```diff
@@ -597,6 +599,27 @@ void debug(ObserverMethod method, boolean interceptor)
       }
    }
 
+   private void debugExtension(Class<?> extension)
```

Comment:
I think that because I have added a method of 3 lines this class is not becoming a god class, or it was a god class and we need to refactor but then we need to decide if we add this refactor in the same issue or create another one or we leave it as is.